plugins/aggregators/histogram/histogram.go

```diff
@@ -90,6 +92,10 @@ var sampleConfig = `
   ## Defaults to true.
   cumulative = true
 
+  ## If true, aggregated histogram are pushed to output only if it was updated since
+  ## previous push. Defaults to false.
+  push_only_on_update = false
+
```

Comment:
Options with a default should be commented like
```suggestion
  ## If true, aggregated histogram are pushed to output only if it was updated since
  ## previous push.
  # push_only_on_update = false

```